packages/synapse-react-client/src/synapse-queries/subscription/useSubscription.ts

```diff
@@ -121,7 +213,6 @@ export const useSubscription = (
     objectId,
     objectType,
   )
-  const { data: subscribers } = useGetSubscribers({ objectId, objectType })
```

Comment:
No need to fetch subscribers in this hook. None of the components that need subscription data also need subscribers data. Reduces network load and prevents some 429s.


This was only used in one place and that one place only used the subscribers data, so that component can just call `useGetSubscribers`.